src/detector.ts

```diff
@@ -71,6 +73,13 @@ function getApiPayload(event: ProductEvent): TopsortEvent {
       id: event.product,
     };
   }
+  let additionalAttribution: Entity | undefined = undefined;
+  if (event.additionalProduct) {
+    additionalAttribution = {
+      type: "product",
```

Comment:
So we're keeping this?

Reply:
@PabloReszczynski

Reply:
for now, as I'm not keen on changing the API

Reply:
Ok, lets keep it then.